dwave_networkx/drawing/chimera_layout.py

```diff
@@ -230,87 +229,86 @@ def draw_chimera(G, **kwargs):
+        If True, each chain in the ``emb`` parameter is labelled with its key.
 
     chain_color : dict (optional, default None)
-        A dict of colors associated with each key in emb.  Should be
-        of the form {node: rgba_color, ...}.  Colors should be length-4
-        tuples of floats between 0 and 1 inclusive. If chain_color is None,
+        Chain colors associated with each key in the ``emb`` parameter as a dict
+        of the form ``{node: rgba_color, ...}``, where colors must be length-4
+        tuples of floats between 0 and 1, inclusive. If None,
         each chain will be assigned a different color.
 
     unused_color : tuple (optional, default (0.9,0.9,0.9,1.0))
-        The color to use for nodes and edges of G which are not involved
-        in chains, and edges which are neither chain edges nor interactions.
-        If unused_color is None, these nodes and edges will not be shown at all.
+        Color to use for graph ``G``'s nodes and edges that are not part of
+        chains, and edges that are neither chain edges nor interactions.
+        If None, these nodes and edges will not be shown.
 
     overlapped_embedding: boolean (optional, default False)
-        If overlapped_embedding is True, then chains in emb may overlap (contain
-        the same vertices in G), and the drawing will display these overlaps as
+        If True, chains in the ``emb`` parameter may overlap (contain
+        the same vertices in ``G``), and the drawing will display these overlaps as
         concentric circles.
 
     kwargs : optional keywords
-       See networkx.draw_networkx() for a description of optional keywords,
-       with the exception of the `pos` parameter which is not used by this
-       function. If `linear_biases` or `quadratic_biases` are provided,
-       any provided `node_color` or `edge_color` arguments are ignored.
+       Parameters in :func:`~networkx.drawing.nx_pylab.draw_networkx`, except for the ``pos`` parameter.
+       If the ``linear_biases`` or ``quadratic_biases`` parameters are specified,
+       the :func:`~networkx.drawing.nx_pylab.draw_networkx` ``node_color``
+       or ``edge_color`` parameters are ignored.
     """
     draw_embedding(G, chimera_layout(G), *args, **kwargs)
 
 
 def draw_chimera_yield(G, **kwargs):
-    """Draws the given graph G with highlighted faults, according to layout.
+    """Draws the given graph ``G`` with highlighted faults.
 
     Parameters
     ----------
     G : NetworkX graph
-        The graph to be parsed for faults
+        :term:`Graph` to be parsed for faults.
 
     unused_color : tuple or color string (optional, default (0.9,0.9,0.9,1.0))
-        The color to use for nodes and edges of G which are not faults.
-        If unused_color is None, these nodes and edges will not be shown at all.
+        Color to use for graph ``G``'s nodes and edges which are not faults.
+        If None, these nodes and edges will not be shown.
```

Comment:
"these nodes and edges will not be shown" --> "these nodes and edges are not shown"